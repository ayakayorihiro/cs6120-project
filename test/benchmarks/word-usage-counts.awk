# Print list of word frequencies
# Arnold Robbins, arnold@gnu.ai.mit.edu, Public Domain
# June 1992

{
    $0 = tolower($0)    # remove case distinctions
    gsub(/[^a-z0-9_ \t]/, "", $0)  # remove punctuation
    for (i = 1; i <= NF; i++)
        freq[$i]++
}

END {
    for (word in freq)
        print word "\t" freq[word]
}
