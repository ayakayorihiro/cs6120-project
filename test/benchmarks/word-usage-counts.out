removed	1
breath	1
morning	6
letter	3
small	2
falling	1
felt	2
grippe	1
folk	2
without	2
automatic	1
do	5
bottom	1
glowed	1
our	1
mission	2
stopped	3
mumbled	1
away	5
out	6
people	1
replaced	1
while	3
moved	3
clouded	1
drip	1
sad	1
fire	5
quite	3
softly	2
interesting	3
salutation	1
became	1
quickly	4
smiled	1
sped	1
day	14
blacked	2
head	5
player	1
shirley	1
addresses	1
private	1
gratefully	1
frayed	1
mother	1
unhappily	1
died	1
feeding	1
hands	2
gorgeous	1
sat	4
irritated	1
trying	3
crash	1
repercussions	1
stiff	1
have	2
twice	1
erect	1
prince	1
saw	3
sharp	1
spied	1
say	2
cleared	1
never	7
uncomfortable	2
well	3
hear	3
chaplain	36
moving	3
fresh	1
pointed	1
yossarian	55
heat	2
extra	1
inquiring	1
possible	1
working	3
1	1
began	2
pinioned	1
affection	1
ibm	1
are	8
gingerly	1
very	9
drone	1
suddenly	1
seemed	2
middleaged	1
dripped	1
declared	2
finest	1
modifiers	1
continued	2
obliterating	1
gazed	2
educated	1
sparse	1
could	9
high	1
woman	3
cases	1
fist	1
dead	3
own	4
cultivating	1
leather	1
groin	1
monotony	1
two	6
exactly	1
thinking	2
medical	1
dear	1
insanity	1
obviously	1
leaving	2
silent	2
pools	1
entered	1
urologist	1
night	3
cried	2
beside	1
remorsefully	1
mary	1
answered	4
sight	1
touched	1
nevertheless	1
who	15
food	1
cid	3
nervously	1
dynamic	1
confessed	1
whispered	2
scampering	1
a	89
ass	1
lingered	1
conversation	1
bombers	1
fled	2
either	2
poem	2
error	1
abrupt	1
go	4
why	3
bent	1
pathos	1
i	44
intense	1
know	11
name	3
him	31
area	1
foot	1
protect	1
rose	3
o	2
rubber	1
elbows	1
stared	1
youre	6
his	62
r	2
went	2
rest	2
some	1
seams	1
pimple	1
covered	1
sighed	2
rim	1
doing	1
insides	1
folkpeople	1
bite	1
broke	1
psychologist	1
member	1
weights	1
god	1
sorry	3
he	99
wont	2
useless	2
exchanged	1
exploded	1
pricks	1
family	1
coupled	1
ashblond	1
with	34
edged	1
maybe	1
between	1
must	5
throughout	1
said	28
talked	2
got	5
duckett	2
annihilating	1
silver	2
time	8
sea	2
possibilities	1
department	1
pedantic	1
unless	1
nose	1
see	6
stifling	1
ceaselessly	1
guess	1
like	6
nest	1
sent	1
atheists	1
pill	2
dwelt	1
insignia	1
hospital	9
id	1
blew	1
caught	3
given	1
if	7
cystologist	1
violet	1
timorously	1
suppose	1
nylon	1
set	1
gives	1
im	9
in	76
liver	4
256th	2
place	1
lives	2
left	3
everyones	1
coughed	2
words	2
circuit	1
partition	1
is	17
thoroughly	1
it	29
hundred	2
patients	2
put	1
full	3
other	8
slightly	1
note	1
homes	1
tab	1
bald	1
excuse	1
days	3
idea	3
whole	4
censoring	2
jar	5
soon	3
turned	2
tiny	1
again	6
tan	1
discharge	1
easy	2
finally	1
conduct	1
grinning	1
what	5
there	17
having	1
mean	2
legs	2
shudders	1
brooklyn	1
speed	1
walk	1
smart	1
tiredly	1
votes	3
censored	1
psyche	1
niggers	2
meat	1
deep	2
brisk	2
centered	1
middle	1
named	1
goofy	1
derogated	1
floated	1
girl	1
grimly	1
everybody	3
friend	1
pads	2
hill	1
limbs	1
efficient	1
were	22
upper	1
cold	3
chair	1
often	1
taken	1
evening	1
still	5
names	1
conciliating	1
thirtytwo	1
word	1
thank	3
characters	1
pumps	1
pleasant	1
those	6
true	1
counted	1
playing	2
ticklish	1
joseph	1
she	7
over	4
cavity	1
kidneys	1
sleep	1
meansdecent	1
moby	1
described	1
anyone	2
since	1
strange	2
drifters	1
loudly	1
minutes	1
children	1
gone	3
did	3
scared	1
shrank	1
quietly	2
fictitious	1
chilled	2
come	5
best	1
craziest	1
same	2
residence	1
artillery	2
dutyeverybody	1
ember	1
message	1
orangetinted	1
cramer	1
inserting	1
white	9
demanded	2
careless	1
ethereal	1
fellows	1
free	1
about	9
pipe	1
higher	1
mosquito	1
want	2
setting	1
crazy	3
talkin	1
himeverybody	1
symbolic	2
supposeeverything	1
told	2
second	1
me	4
going	1
father	2
inefficient	1
above	1
madly	1
good	12
three	4
kept	3
romance	1
both	1
tensions	1
soldier	7
found	1
sane	2
communist	1
crashed	1
four	2
likable	1
flicks	1
distaste	1
strangely	1
my	4
anxiety	1
hot	3
drove	1
how	2
organ	1
minded	1
returning	1
ted	1
next	6
apologized	1
death	1
lieutenant	7
under	1
embarrassed	1
tepid	1
safely	1
something	2
when	13
signatures	1
cysts	1
ten	2
collar	2
slim	1
managed	1
slip	1
lips	3
lunatics	1
lowering	1
but	20
no	21
specialists	1
color	1
probably	2
another	5
serious	4
through	3
machine	1
yes	12
mouth	4
shanghaied	1
remained	1
sends	1
meticulously	1
thermometers	1
afraid	1
motionless	1
summer	2
mps	1
give	3
fighting	5
regards	1
harvard	1
read	5
of	65
far	3
oh	1
hoses	1
attacking	1
salutations	1
novel	1
fighter	3
assured	1
along	2
stately	1
on	26
across	1
faintly	1
confused	1
filled	1
dubiously	1
whores	1
or	5
hips	1
aisle	1
power	1
untouched	1
murdered	2
where	1
matriotism	1
mustache	1
ill	3
books	1
madman	1
fluid	1
just	7
endocrinologist	1
havent	2
roar	1
indecent	1
captains	1
wasnt	4
concealing	1
pail	1
murderer	1
stuff	1
anchored	1
quiet	1
solemn	1
pain	5
took	2
arms	2
myself	1
agreed	1
forced	1
shoved	1
milk	1
unspringing	1
ward	18
army	1
call	1
bars	1
hole	2
chess	3
help	2
curiosity	1
games	3
blaze	2
elbow	1
ecstatically	1
patient	2
question	1
simply	1
comfortable	3
take	1
stirred	1
bandages	1
neat	1
everyone	2
universal	1
spoke	2
floor	3
views	1
impatiently	1
heels	1
eight	1
enlistedmen	1
base	1
relief	1
the	266
world	2
reached	1
determine	1
proscribing	1
sympathetic	1
irving	3
came	3
suspected	2
belly	1
apple	1
actually	1
wouldnt	1
long	1
chocolate	1
cigarettes	1
than	6
instant	1
colored	1
boredom	1
eyes	6
anode	1
wearing	1
learn	1
derma	1
warned	1
that	42
idly	1
appropriate	1
sewn	1
encased	1
thats	13
tall	1
dying	1
arrived	2
much	2
flashed	1
worked	1
first	7
nately	4
mind	1
someone	2
should	1
temperature	2
frantic	1
made	3
bedside	1
foolish	1
busy	2
cracks	1
weighed	1
astonished	1
dog	1
articles	1
sick	4
disturbed	1
touch	2
look	1
group	3
juice	1
during	2
hair	2
hoisted	1
flushed	1
envelopes	2
yossarians	6
section	1
fed	1
hurled	1
least	1
pale	1
spells	1
completed	1
slowly	1
sound	1
knees	1
airfield	1
rations	1
palm	1
not	12
pneumonia	1
explained	1
now	3
loony	1
perpendicularly	1
side	7
jaundice	5
few	1
carefully	1
eighth	1
special	1
humanly	1
wheeled	1
missingall	1
every	6
so	3
ruthlessly	1
placed	2
nurse	4
grew	1
transmitting	1
miles	1
spend	2
shipman	2
black	1
discuss	1
shit	1
each	9
expurgating	1
bowels	1
cetologist	1
stood	1
sorrow	1
committing	1
bad	7
times	1
enlisted	1
smile	2
drawl	1
sergeant	2
visited	2
messages	1
careful	1
letters	9
spent	1
slender	1
rather	1
isnt	4
better	1
washington	3
dont	2
patriotically	1
dizzy	1
doctors	6
bear	1
brown	1
need	2
nurses	4
lymph	1
invented	1
field	1
theyre	2
to	61
contagious	1
plane	1
sessions	1
gentle	1
killed	2
chart	1
irritation	1
lay	1
zinc	1
old	1
discovered	1
corps	1
plyboard	1
startled	1
suspiciously	1
killer	1
noticed	1
military	1
beamed	1
posing	1
returned	1
always	3
flaming	1
spines	1
echelon	1
dusted	1
shyly	1
missingand	1
half	2
shirt	1
shattered	1
mentioning	1
hall	1
waste	1
sideways	1
folkshould	1
catch22	2
up	10
health	1
among	1
them	7
after	8
lights	1
which	2
degenerates	1
then	9
us	2
parts	1
started	1
urine	1
sooner	1
balanced	1
dolls	1
they	24
golden	1
please	2
carried	1
paper	1
metropolises	1
inevitably	1
wrist	1
here	1
meals	2
passed	1
fellas	3
inquire	1
everything	5
bolted	1
appeared	1
texan	14
dunbar	21
officer	6
squadron	4
ancient	1
less	2
texas	1
stay	1
plundered	1
end	1
right	5
actions	1
write	1
firemen	4
cement	1
goodnatured	1
adverb	1
windows	1
forward	1
didnt	12
chaplains	3
feel	2
one	15
colonel	9
faithfully	1
moms	1
volunteers	1
landed	1
reply	1
air	1
heller	1
from	20
pie	1
empty	1
around	6
clear	4
table	1
feet	2
hand	2
theres	1
grin	2
case	2
ripple	1
following	1
yearn	1
beds	2
hang	1
anabaptist	1
plaster	1
apart	1
lying	3
knew	5
watered	1
stand	1
shook	2
softer	1
pianosa	2
shouted	1
hose	1
lymphologist	1
ceiling	2
job	1
until	1
before	4
break	1
peered	1
choice	1
bed	12
body	2
fascinated	1
pressed	1
slight	2
glutinous	1
heavily	2
grave	1
exodus	1
puzzled	1
elba	1
excitedly	1
nigger	1
filed	1
its	2
erected	1
decent	2
toe	1
caution	1
visit	4
diffident	1
really	5
been	11
stooped	1
efficiently	1
means	1
meant	2
smoke	1
wanted	2
whom	1
recorded	1
alarm	1
101	1
persisted	1
troubling	1
screw	1
trucks	2
too	8
whos	3
get	3
saying	1
written	2
himself	2
creativity	1
dredged	1
straight	1
stoppered	1
narrow	1
though	1
square	1
brought	3
monotonous	4
officers	2
attention	1
distributing	1
temperatures	1
you	31
accord	1
straightened	1
anything	4
patriotism	2
investigated	1
dresses	1
thermometer	4
nerves	1
exhausted	1
censor	3
making	1
had	39
mouths	1
south	1
text	1
make	2
can	3
generous	1
staring	1
telling	1
entire	2
love	2
part	1
endocrines	1
planes	2
timber	1
inches	1
ive	4
thin	1
wac	1
basin	1
annoyance	1
erica	1
has	1
man	5
shot	3
photographed	1
seen	1
show	1
rhythms	1
face	3
basis	1
this	6
doctor	1
ran	1
nodded	1
into	12
matter	1
lower	1
all	25
intralinear	1
delivered	1
fever	1
hour	1
may	2
expired	1
dermatologist	1
completely	1
because	6
become	2
war	2
promised	1
mental	1
was	74
fact	3
midwinter	1
innocent	1
almost	1
zoology	1
necessary	1
way	1
accommodate	1
pathologist	1
epitome	1
past	1
incident	1
raced	1
toys	1
little	1
wards	5
drink	1
replied	1
thought	1
behind	1
arent	1
blood	1
granted	1
zippered	1
fighterpilot	1
regular	1
short	2
communications	1
regretfully	1
roll	1
fifteen	1
stove	1
cross	2
drew	1
silence	1
darkly	1
needles	1
produced	1
tomorrow	1
gauze	2
untroubled	1
back	9
adriatic	1
allow	1
adjective	1
span	2
enjoyed	1
hard	2
am	1
coffee	1
cheeks	1
resume	1
an	11
movement	1
interior	1
fight	2
criminals	1
think	1
technicolor	1
disappointed	3
as	18
your	1
switched	1
at	25
twentyfouryearold	1
mutual	1
fruit	1
island	1
more	7
last	2
heard	2
tragically	1
sweetfaced	1
sharply	1
and	114
dodgers	1
pastel	1
even	10
their	5
upon	1
suspended	1
sealed	1
writing	2
afternoon	5
ever	4
whispering	1
knuckles	1
big	1
start	1
green	1
be	5
kitchen	1
captain	15
unimpressed	3
keep	2
wrote	6
drugged	1
cheek	1
late	1
exclaimed	1
uncertain	1
dangerous	2
any	1
curly	1
bit	2
rammed	1
vortex	1
order	1
sweet	1
smuggled	2
voice	1
awoke	1
admit	1
banged	1
by	12
mess	1
asked	10
giggle	1
dabbed	1
feeling	1
many	1
increasing	2
being	2
added	2
ahead	1
malaria	1
conscience	1
down	13
early	1
others	1
most	2
cavernous	3
mildewed	1
couldnt	1
fingered	1
for	22
accompanied	1
lies	1
amorously	1
only	6
dick	1
nothing	1
liked	1
dedicated	1
streets	1
red	1
fidgeted	1
roof	1
her	3
donated	1
served	1
hes	2
pretty	3
men	7
faulty	1
mediterranean	1
specializing	1
none	2
treat	1
met	1
lead	1
life	2
warrant	4
kill	1
looked	3
fell	2
required	1
